src/vaadin-grid-column-reordering-mixin.html

```diff
@@ -167,7 +167,23 @@
       const targetColumn = this._getTargetColumn(targetCell, this._draggedColumn);
       if (this._isSwapAllowed(this._draggedColumn, targetColumn) &&
         this._isSwappableByPosition(targetColumn, e.detail.x)) {
-        this._swapColumnOrders(this._draggedColumn, targetColumn);
+        // Get the column header level of the target column (and the dragged column)
```

Comment:
Small fix to the comment done [here](https://github.com/vaadin/web-components/commit/114012d63f341c242d15cc7e14a9de5920a83f27#diff-20a3e04168d572628bd90070c5e06bc4c807a676a992e2c9bf08f5dfdc5921b5L164-R164)

```suggestion
        // Get the header level of the target column (and the dragged column)
```